research/models.py

```diff
@@ -0,0 +1,18 @@
+from django.db   import models
+
+class research_information(models.Model): 
```

Comment:
클래스 명은 대문자로 시작해야됩니다. 제가 아까 혼동을 드린것 같네요.

```suggestion
class ResearchInformation(models.Model): 
```

Reply:
아하 처음에 했던 걸로 다시 해서 푸쉬해서 올리겠습니다!

Reply:
네 아까 바껴야될 부분이  `db_table` 이라고 언급했어야 되었는데, 언급을 하지 않았네요.